src/views/v2/list/nav/next-disabled.php

```diff
@@ -9,19 +9,20 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.10
+ * @version TBD
  *
  */
 ?>
 <li class="tribe-events-c-nav__list-item tribe-events-c-nav__list-item--next">
-	<button class="tribe-events-c-nav__next tribe-common-b2 tribe-common-b1--min-medium" disabled>
-		<?php
-			$events_label = '<span class="tribe-events-c-nav__next-label-plural"> ' . tribe_get_event_label_plural() . '</span>';
-			echo wp_kses(
-				/* translators: %s: Event (plural or singular). */
-				sprintf( __( 'Next %1$s', 'the-events-calendar' ), $events_label ),
-				[ 'span' => [ 'class' => [] ] ]
-			);
-		?>
-	</button>
+	<button 
+		class="tribe-events-c-nav__next tribe-common-b2 tribe-common-b1--min-medium" 
+		disabled
+	><?php
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 4101a1f40776af81a0c171c934e1e869</sub>